### DISCUSSION TITLE
yall are gonna have to give me a whole ass video explaining how rigtools works.

### DISCUSSION THREAD

**Discussion by rigger59** [3kh0/ext-remover#1261] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
wtf is "extension id" I couldn't find that

**Comment by Sam-hacks1** [comment]
fr

**Comment by schoolexploitkid** [comment]
:v

**Comment by HelloIloveGimkit2** [comment]
The fucking extension ID is the thing u see when you go to chrome://extensions why don't you stop being a dumbass
![image](https://github.com/user-attachments/assets/617e359a-5570-4bcf-8cea-88d12c6fd95e)
![image](https://github.com/user-attachments/assets/c3f573db-9149-4ff7-b496-56675f7aac17)
![image](https://github.com/user-attachments/assets/41a51801-b397-4786-9165-7a24edc8a764)

**Reply by rigger59** [reply]
where do we paste it in tho?

**Reply by HelloIloveGimkit2** [reply]
IN THE FUCKING EXTENSION ID BOX WHEN YOU FUCKING CLICK THE EXTENSION ID THINGY YOU PASTE THE FUCKING ID IN ARE YOU FUCKING RETARDED OR WHAT

**Reply by rigger59** [reply]
> IN THE FUCKING EXTENSION ID BOX WHEN YOU FUCKING CLICK THE EXTENSION ID THINGY YOU PASTE THE FUCKING ID IN ARE YOU FUCKING RETARDED OR WHAT

give me pictures of the process please.

**Reply by HelloIloveGimkit2** [reply]
FUCKING DO IT URSELF THERE IS NO VIDEO AND THEN ALSO I'M ON V129 CHECK UR VERSION BECAUSE I WILL NOT FUCKING DEAL WITH YOUR SHIT AND YOU CAN'T EVEN DO IT

**Reply by HelloIloveGimkit2** [reply]
alright idiot terms if ur on v129 or higher you can't do the exploit so check ur version before doing it

**Reply by HelloIloveGimkit2** [reply, marked as answer]
here's the steps broken down I don't have a fucking video
![image](https://github.com/user-attachments/assets/a2553b56-4446-47ce-ac1e-6179f4e29ece)
![image](https://github.com/user-attachments/assets/f7a2d61a-627e-46d4-ba6c-09a83a998a3b)

**Reply by HelloIloveGimkit2** [reply]
also please don't try unenrolling using brick cuz if you can't follow basic instructions little timmeh then don't try br1ck cuz it's big boy stuff

**Reply by HelloIloveGimkit2** [reply]
this repo is so skid friendly that I don't know how some of y'all fuck up this badly

**Reply by rigger59** [reply]
dw i got it now you good

**Comment by rigger59** [comment]
[Uploading here.webm…]()
answer here
